my-app/src/components/specialisation/Specialisation.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from 'react'
+import './specialisation.css';
+import {useState} from "react";
+
+const Specialisation = (props) => {
+    const userInfo = {
+          eng_major: '',
+    }
+      
+    const [major, setMajor] = useState('');
+
+    const test = useState('');
+      userInfo.eng_major = major;
+
+
+
+    const handleClick = () => {
+      }
+      
+  const makeLayout = () => {
+    
+    return (
+      <div className="search-container">
+        
+            <div>
+                <h4 className="select-course"> Select your relevant course</h4>
+                <h5 className="current-selection">Currently selected: {major} </h5>
+            </div>
+          
+            <div>
+                <div className="semester-block">
```

Comment:
similar to before, only save the user selection on click of the submit button -> not everytime they click on a specialisation